ui/components/src/ui-components/tree/controlled/component/ControlledTree.tsx

```diff
@@ -23,7 +23,7 @@ import { RenderedItemsRange, TreeRenderer, TreeRendererProps } from "./TreeRende
 
 /**
  * Properties for [[ControlledTree]]
- * @beta
```

Comment:
Last time we discussed this we said that the promise to keep `@public` APIs stable for 2 major releases goes into effect starting with 3.0, so it would be acceptable to change a public 2.0 API in 3.0.

Reply:
ok, that sounds good, thanks